### PR TITLE
fix: imagenette example works again

### PR DESCRIPTION
**Pull request by xnought**
also very fast now, inference on 2 resnet models take ~12 min compared to ~1 hour for me before
imagenette

### CODE REVIEW

**Comment by xnought** [comment]
Speedup is because I removed gradient accumulation with @torch.no_grad() and I kept the argmax operation within torch instead of using numpy.

**Review by cabreraalex — APPROVED** [review]

**Comment by cabreraalex** [comment]
EPIC
